cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterInventory.tsx

```diff
@@ -1,32 +1,72 @@
 // React imports
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 // Library imports
 import { CdsButton } from '@cds/react/button';
 import { CdsIcon } from '@cds/react/icon';
+import { ClarityIcons, disconnectIcon } from '@cds/core/icon';
+import { CdsAlert, CdsAlertGroup } from '@cds/react/alert';
+
+// App imports
+import { NavRoutes } from '../../shared/constants/NavRoutes.constants';
+import { UnmanagedCluster } from '../../swagger-api';
+import { UnmanagedService } from '../../swagger-api/services/UnmanagedService';
+import UnmanagedClusterInfo from '../../shared/components/UnmanagedClusterInfo/UnmanagedClusterInfo';
+import './UnmanagedClusterInventory.scss';
+
+ClarityIcons.addIcons(disconnectIcon);
 
 const UnmanagedClusterInventory: React.FC = () => {
```

Comment:
Please use function keyword for the function definition.

Reply:
Oops. we missed that one. Could you please correct it in the PR? 
When we just started the project, we were using both ways to create components. But later on we want to make it consistent and decided to use function keyword in the component creation. 
Arrow function is a new way to create a function in es6. There are some differences between regular function and arrow function: https://dmitripavlutin.com/differences-between-arrow-and-regular-functions/

Reply:
Will do!